.env

```diff
@@ -12,12 +12,12 @@ POSTGRES_PASSWORD=password
 
 # Docker Image tags
 MANAGEMENT_TAG=beta001
-MANAGEMENT_BASE_TAG=latest
-BLACKLIGHT_BASE_TAG=105ac
```

Comment:
It doesn't seem like the management app should need to be concerned with the BLACKLIGHT_BASE_TAG.  But that's a topic for  a different PR.